integration/monai/examples/spleen_ct_segmentation/README.md

```diff
@@ -53,6 +53,15 @@ and assign them to the appropriate GPU device using the `--gpu` argument.
 python3 -u -m nvflare.private.fed.app.simulator.simulator job --workspace /tmp/nvflare/sim_spleen_ct_seg --threads 2 --clients site-1,site-2 --gpu 0,1
 ```
 
+#### 4.3 Single thread, multi-gpu training
+If you have several gpus in your system and want to simulate multi-gpu training on one client, 
+please follow step 2 & 3 above but replace `job` with `job_multi_gpu`. This will use NVFlare's `PTMultiProcessExecutor` 
+to start multi-gpu training using [torchrun](https://pytorch.org/docs/stable/elastic/run.html) on one client. 
+
+```
+python3 -u -m nvflare.private.fed.app.simulator.simulator job_multi_gpu --workspace /tmp/nvflare/sim_spleen_ct_seg --threads 1 --n_clients 1
```

Comment:
Another thing here, people might get confused by "Single thread, multi-gpu training"

Reply:
Maybe best to remove "Single thread" here. The example is mainly about simulating multi-gpu training using 1 client.